instructure/instructure-ui
# docs(ui-top-nav-bar): add breadcrumb example for topnavbar

INSTUI-3915

test plan:

- go to the docs and open the TopNavBar playground
- enable/disable the `inverseColor` and `showBreadcrumb` prop (make sure to other conditions are met, e.g.: brand is not shown)
